### ISSUE TITLE
How is the HRSCD dataset split?

### ISSUE THREAD

**Issue by dizi-oss** [zhuduowang/Change3D#14]
<img width="309" height="439" alt="Image" src="https://github.com/user-attachments/assets/37a283c0-33b1-4f13-ab89-c96c83ebf494" />
Dear authors, I downloaded the HRSCD dataset from the official website as shown in the figure above, but the site does not provide a concrete training / testing / validation split. Could you kindly share the corresponding splitting code or any recommendations? Thank you very much!

**Comment by LINGQI711** [comment]
I am unable to reproduce the results in the paper, apparently because there are too few change areas in HRSCD and I have learned too much about 'No info'. How were the parameters or loss functions set for the results in the paper? Thank you very, very much!
